ddocs/medic-db/medic-client/views/reports_by_freetext/map.js

```diff
@@ -15,32 +18,33 @@ function(doc) {
     if (!key || !value) {
       return;
     }
-    key = key.toLowerCase();
-    if (skip.indexOf(key) !== -1 || /_date$/.test(key)) {
-      return;
-    }
+
     if (typeof value === 'string') {
       value = value.toLowerCase();
       value.split(/\s+/).forEach(function(word) {
         emitMaybe(word, reportedDate);
       });
     }
-    if (typeof value === 'number' || typeof value === 'string') {
-      emitMaybe(key + ':' + value, reportedDate);
-    }
   };
 
-  if (doc.type === 'data_record' && doc.form) {
-    Object.keys(doc).forEach(function(key) {
-      emitField(key, doc[key], doc.reported_date);
-    });
+  var include = ['patient_id', 'patient_uuid', 'place_id', 'case_id', 'patient_name'];
```

Comment:
> `patient_uuid`

Is there a specific reason to include this field? In the discussion on the issue thread, it was noted that there is probably minimal value in indexing UUID values (since who is going to actually search for those)

Reply:
You're right, I wrongly assumed `patient_id` and `patient_uuid` represented the same thing :sweat_smile: